site/content/en/docs/technical-outcomes/scalability/scale-using-hpa/_index.md

```diff
@@ -0,0 +1,215 @@
+---
+title: "Horizontal Pod Autoscaling"
+---
+
+# Horizontal pod autoscaling
+
+Using KWOK we can deploy a metric server to help us observe and trigger our HorizontalPodAutoscaler.
+
+This image shows you what you should expect when testing this scenario.
+You can follow the step-by-step guide after seeing this.
+
+<img width="700px" src="hpa.svg">
```

Comment:
Hi @wzshiming, I keep getting this error after executing the make command to generate svg.

```bash
make site/content/en/docs/technical-outcomes/scalability/scale-using-hpa/hpa.svg

make: *** No rule to make target 'site/content/en/docs/technical-outcomes/scalability/scale-using-hpa/hpa.svg'.  Stop.
```

Reply:
Because there is no `hpa.demo` in this directory, it is `scale-using-hpa.demo`.

```
make site/content/en/docs/technical-outcomes/scalability/scale-using-hpa/scale-using-hpa.svg
```

Reply:
Nice